apps/whale-api/src/module.indexer/model/dftx/create.pool.pair.ts

```diff
@@ -37,6 +37,123 @@ export class CreatePoolPairIndexer extends DfTxIndexer<PoolCreatePair> {
     super()
   }
 
+  async indexBlockEnd (block: RawBlock): Promise<void> {
+    // {
+    //   "90": {
+    //     "symbol": "AMZN-DUSD",
+    //     "name": "dAMZN-Decentralized USD",
+    //     "status": true,
+    //     "idTokenA": "89",
+    //     "idTokenB": "15",
+    //     "dexFeePctTokenA": 0.00100000,
+    //     "dexFeeInPctTokenA": 0.00100000,
+    //     "dexFeePctTokenB": 0.00100000,
+    //     "dexFeeOutPctTokenB": 0.00100000,
+    //     "dexFeeInPctTokenB": 0.00100000,
+    //     "dexFeeOutPctTokenA": 0.00100000,
+    //     "reserveA": 23610.96677637,
+    //     "reserveB": 2970783.28265054,
+    //     "commission": 0.00200000,
+    //     "totalLiquidity": 264845.36091854,
+    //     "reserveA/reserveB": 0.00794772,
+    //     "reserveB/reserveA": 125.82217876,
+    //     "tradeEnabled": true,
+    //     "ownerAddress": "8UAhRuUFCyFUHEPD7qvtj8Zy2HxF5HH5nb",
+    //     "blockCommissionA": 0.00000000,
+    //     "blockCommissionB": 0.00000000,
+    //     "rewardPct": 0.00000000,
+    //     "rewardLoanPct": 0.02230000,
+    //     "creationTx": "b929750896550d830ce9a44485a256741d3fae9abf5db289e87934b920767807",
+    //     "creationHeight": 1948150
+    //   }
+    // }
+
+    if (block.height === 1948150) {
+      console.log('Creating dAMZN-DUSD')
+      // https://defiscan.live/tokens/dAMZN-DUSD
+      await this.poolPairHistoryMapper.put({
+        id: 'b929750896550d830ce9a44485a256741d3fae9abf5db289e87934b920767807',
+        sort: HexEncoder.encodeHeight(block.height) + HexEncoder.encodeHeight(0),
+        poolPairId: '90',
+        pairSymbol: 'AMZN-DUSD',
```

Comment:
hmm.. do we need update the old poolpair in index as well??

Reply:
Let me create another ticket, to move away from indexing a few DfTx

Reply:
https://github.com/JellyfishSDK/jellyfish/issues/1544